src/main/java/fi/otavanopisto/kuntaapi/server/controllers/IdentifierController.java

```diff
@@ -71,9 +66,29 @@ public Identifier findIdentifierByTypeSourceAndKuntaApiId(IdType type, String so
     return findIdentifierByTypeSourceAndKuntaApiId(type.toString(), source, kuntaApiId);
   }
 
-  public List<Identifier> listIdentifiersByTypeAndSource(IdType type, String source, Integer firstResult, Integer maxResults) {
-    return identifierDAO.listByTypeAndSource(type.toString(), source, firstResult, maxResults);
+  private Identifier createIdentifier(String type, String kuntaApiId, String source, String sourceId, String organizationKuntaApiId) {
+    return identifierDAO.create(type, kuntaApiId, source, sourceId, organizationKuntaApiId);
   }
 
+  private Identifier findIdentifierByTypeSourceIdAndOrganizationId(String type, String source, String sourceId, String organizationKuntaApiId) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "findIdentifierByTypeSourceIdAndOrganizationId" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUnusedPrivateMethod)